src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/NuGetReference.cs

```diff
@@ -18,5 +19,7 @@ public override string ToString()
         }
 
         public string? PrivateAssets { get; set; }
+
+        public IEnumerable<string>? ActionDetails { get; set; }
```

Comment:
Can you make this non-nullable? Is there a reason you want this nullable?

Reply:
Can you make this (and the other properties while you're at it) `get; init;` instead of `set;`?

Reply:
I'm also confused as to why this is here. Why is this on this object? I'm not understanding why a NuGetReference would need to have an action details